index.html

```diff
@@ -12,22 +12,27 @@
 
     <body>
         <input class="search-box" type="text" oninput="handleChange(this)" placeholder="search"/>
-        <div id="cards-holder"></div> 
+        <section id="cards-holder" class="card-list" ></section>
```

Comment:
Can you check if it's possible to make it scroll downwards? I think it may look better